sdk/python/feast/job_service.py

```diff
@@ -145,6 +167,54 @@ def GetJob(self, request, context):
         return GetJobResponse(job=self._job_to_proto(job))
 
 
+def run_control_loop():
+    """Runs control loop that continuously ensures that correct jobs are being run."""
+    logging.info(
+        "Feast Job Service is starting a control loop in a background thread..."
```

Comment:
This log is a bit vague? What do I as a a Feast admin understand from this?

Reply:
Suggestion?